.changeset/purple-pumas-study.md

```diff
@@ -0,0 +1,18 @@
+---
+'gatsby-plugin-clerk': major
+'@clerk/chrome-extension': major
+'@clerk/clerk-js': major
+'@clerk/clerk-sdk-node': major
+'@clerk/backend': major
+'@clerk/fastify': major
+'@clerk/nextjs': major
+'@clerk/shared': major
+'@clerk/themes': major
+'@clerk/clerk-react': major
+'@clerk/remix': major
+'@clerk/types': major
+'@clerk/clerk-expo': major
+---
+
+Upgrade all packages to node@v18 version, moved some deps to dev dependencies
+and introduced `isomorphicBtoa` helper in `@clerk/shared`.
```

Comment:
```suggestion
Dropping support for Node 14 and 16 as they both reached EOL status. The minimal Node.js version required by Clerk is `18.18.0` now.
```

For the new helper in `@clerk/shared` you can add a new changeset with a `minor`